### DISCUSSION TITLE
Accommodate DragonOS Focal (20.04) in installer

### DISCUSSION THREAD

**Discussion by alphafox02** [ainfosec/FISSURE#11] [General]
I’d like to take a stab as I mentioned earlier at including FISSURE into DragonOS as it contains nearly or most of the required tools already to include 3.8.1 and a good majority of the OOT modules. It also already has all drivers setup in a specific way etc. 

I think maybe the sustainable way to accommodate is by adding to the installer script a DragonOS option, then installing only the missing pieces while trying to avoid breaking changes.

**Comment by cpoore1** [comment]
Sure, we can do whatever you'd like. I imagine most of FISSURE will play nice with what's going on, especially if you were able to launch the Dashboard just fine. I have no problem treating the install as just another OS and then adding a few if statements in the code to point towards new locations for certain programs. I'm sure you have a good amount of experience when it comes to determining how much software you want people installing on their own. 

If you'd like you can email me (see README) and we can talk about the specifics. Send me a list of programs that we have in common that you don't want me to duplicate and I can verify all the functionality within DragonOS.

**Reply by alphafox02** [reply]
Might take me a little time for the list, but I’ll work on it. From what I can tell the core that’s needed for the dashboard is already there, to include dependencies, almost all drivers, most OOT modules, Falcon, etc. All other more Kali like tools, Airgeddon etc are not installed. I just need to get the energy to go line by line and compare. :)
